code/modules/events/hostile_infestation.dm

```diff
@@ -0,0 +1,125 @@
+//Hostile Infestation: A variant of the infestation event that spawns small numbers of hostile mobs.
+//Currently only fires if there are 2 or more security officers
+
+#define LOC_KITCHEN 0
```

Comment:
I think a lobotomized chimpanzee is probably better at coding than me at this point. If what I'm trying to do can't be stolen and repurposed from other areas of the code, I can't do it.